corecito.py

```diff
@@ -47,26 +48,38 @@ async def main():
       decrease_percentage = missing * 100 / account.core_number
 
       if coreNumberExploded(account.core_number, deviated_core_number, account.max_core_number_increase_percentage):
-        logger.info(f'> Exploded {increase_percentage:.2f}%\nConsider updating CoreNumber to {deviated_core_number:.6f}')
+        log_message = f'> Exploded {increase_percentage:.2f}%\nConsider updating CoreNumber to {deviated_core_number:.6f}'
+        logger.info(log_message)
+        if config['telegram_notifications_on']:
+          telegram_bot_sendtext(config['telegram_bot_token'], config['telegram_user_id'], log_message)
```

Comment:
The log messages are beginning to make the code unreadable, especially when this and #8 are merged, there will be way more logging lines than code in these functions.
What do you think about moving all these 4 lines to a separate function like `logPriceExploded(...)`?